pkg/yurthub/cachemanager/cache_manager.go

```diff
@@ -91,8 +105,11 @@ func NewCacheManager(
 		restMapperManager:     restMapperMgr,
 		listSelectorCollector: make(map[storage.Key]string),
 		inMemoryCache:         make(map[string]runtime.Object),
+		errorKeys:             NewErrorKeys(),
```

Comment:
Cache Manager is used for managing cache data of Yurthub, so it is not a good idea to put autonomy condition handling in the CacheManager, how about add a new http.Handler for dealing with autonomy condition setting? At the same time `CacheManager` or `Storage Wrapper` maybe used by this new handler.

Reply:
Makes sense, I just introduced an autonomy proxy to update node conditions.